repose-aggregator/components/filters/api-validator-filter/src/main/java/org/openrepose/filters/apivalidator/ApiValidatorFilter.java

```diff
@@ -43,15 +44,15 @@
     private static final String SCHEMA_FILE_NAME = "/META-INF/schema/config/validator-configuration.xsd";
 
     private final ConfigurationService configurationService;
-    private final MetricsService metricsService;
+    private final Optional<MetricsService> metricsService;
```

Comment:
I should note that IntelliJ whines about having `Optional` fields and constructor parameters. Not a big deal, but something to mention.